cryptoservice/crypto_service.go

```diff
@@ -82,10 +82,15 @@ func (cs *CryptoService) GetPrivateKey(keyID string) (k data.PrivateKey, role st
 	for _, ks := range cs.keyStores {
 		for _, keyPath := range keyPaths {
 			k, role, err = ks.GetKey(keyPath)
-			if err != nil {
+			if err == nil {
+				return
+			}
+			switch err.(type) {
+			case trustmanager.ErrPasswordInvalid, trustmanager.ErrAttemptsExceeded:
```

Comment:
Given that the keyStores are stored and iterated over in order of usage preference, I thought might failing on the first key would make sense?  I was mentally mapping this to something like multiple 2FA devices, where it asks you for your preferred 2nd factor token, and only if you explicitly say "I don't have that factor, please use a different one" does it ask you for another.

We also explicitly recommend the user remove their on-disk key once the key has already been imported into the yubikey, so this seems reasonable unless we explicitly decide to support/encourage the general case the same key in multiple keystores?